include/swift/SIL/MemAccessUtils.h

```diff
@@ -1558,14 +1584,37 @@ class AccessUseDefChainCloner
   }
 };
 
-template <typename UnaryPredicate>
+/// Clone all projections and casts on the access use-def chain until the
+/// checkBase predicate returns a valid base.
+///
+/// This will not clone ref_element_addr or ref_tail_addr because those aren't
+/// part of the access chain.
+///
+/// CheckBase is a unary predicate that takes the next source address and either
```

Comment:
This is the comment on CheckBase that I wanted above. Maybe add a note above to point here for more information about CheckBase?

Reply:
I had all these comments duplicated, decided the was a bad idea, so added `See comments on the cloneUseDefChain() API.`